### PR TITLE
DAF-4946: "Upcoming third-party SDK requirements" 

### PR DESCRIPTION
**Pull request by mvn-hieunguyen-hn**
## Description
[iOS] "Upcoming third-party SDK requirements" 
### Solution
Change wakelock to wakelock_plus
### What was done (Please be a little bit specific)
Change wakelock to wakelock_plus
### JIRA
https://dw-ml-nfc.atlassian.net/browse/DAF-4946

## Screenshot or Video (Before/After)
(For a new design, a screenshot should be required. But for new functionality, a video is a great help.)

## Ready for review checklist
Basically, you must check all. (When you ignore any checks, please write each reason.)

- [x] All sections above are properly filled in
- [x] The PR only deals with one functionality/bug
- [ ] Includes code refactoring? If yes, the following sub checks are required.
    - [ ] The changes were tested. So there is no degradation.
    - [ ] The change is not too big. (Reviewers can review it.)
- [ ] Includes change of spec? If yes, the sub check is required.
    - [ ] Updated the document.
- [x] Builds and runs on iOS (No new warnings nor new errors)
- [x] Builds and runs on Android (No new warnings nor new errors)

### CODE REVIEW

**Review by mvn-thanhluu-hn — APPROVED** [review]
LGTM.

@noboru-i 
Please review this PR 🙏

File: pubspec.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ dependencies:
   flutter:
     sdk: flutter
   cupertino_icons: ^1.0.5
-  wakelock: ^0.6.1+2
+  wakelock_plus: ^1.2.4
```
Comment (noboru-i) [inline pubspec.yaml]: Sorry...
wakelock_plus 1.2.5 is released 4 days ago.
https://pub.dev/packages/wakelock_plus/changelog

Could you update it? 🙏

**Review by noboru-i — COMMENTED** [review]
@mvn-hieunguyen-hn 
Sorry, please check my comment 🙏

**Review by noboru-i — APPROVED** [review]
LGTM 👍

**Pull request merged**